Description: Skip VersalDeformations test that doesn't catch alarm on mips64el
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1392
Last-Update: 2020-12-04

--- a/M2/Macaulay2/packages/VersalDeformations.m2
+++ b/M2/Macaulay2/packages/VersalDeformations.m2
@@ -1369,7 +1369,6 @@
 S = ZZ/32003[a..d]
 I = monomialIdeal(a^2,a*b,b^4,a*c^3)
 J = truncate(3, I)
-(F,R,G,C) = localHilbertScheme(gens J, Verbose=>4,DegreeBound=>1,HighestOrder=>8)
 ///
 TEST ///
 S=QQ[x,y]/ideal {x^4+y^3}
